src/SprykerEco/Zed/Optivo/Dependency/Facade/OptivoToSalesFacadeBridge.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+namespace SprykerEco\Zed\Optivo\Dependency\Facade;
+
+use Generated\Shared\Transfer\OrderTransfer;
+
+class OptivoToSalesFacadeBridge implements OptivoToSalesFacadeInterface
+{
+    /**
+     * @var \Spryker\Zed\Sales\Business\SalesFacadeInterface
+     */
+    protected $salesFacade;
+
+    /**
+     * @param \Spryker\Zed\Sales\Business\SalesFacadeInterface $salesFacade
+     */
+    public function __construct($salesFacade)
+    {
+        $this->salesFacade = $salesFacade;
+    }
+
+    /**
+     * @param int $idOrder
+     *
+     * @return \Generated\Shared\Transfer\OrderTransfer
+     */
+    public function getOrderByIdSalesOrder(int $idOrder): OrderTransfer
```

Comment:
Please use exactly the same method typehinting like in SalesFacadeInterface.

Reply:
are you sure?